teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -122,6 +125,17 @@ public void configure(TekuConfiguration.Builder builder) {
                     convertToPath(validatorExternalSignerTruststorePasswordFile)));
   }
 
+  public void configure(
+      final TekuConfiguration.Builder builder, final ValidatorClientDataOptions dataOptions) {
+    final DataDirLayout dataDirLayout = DataDirLayout.createFrom(dataOptions.getDataConfig());
+    validatorKeys =
```

Comment:
I'm not super convinced here.
If I understand correctly you are "emulating" a validatorKey option.
I'd expected to have the ability to exit all kind of keys by specifing the new param `--validator-public-keys`.

So if you specify  `--data-validator-path`, `--validator-keys` and `--validator-public-keys` the command loads all managed keys and exit only those matching the provided pub keys.
This way you could also exit a specific pub key by providing `--validator-keys` pointing to a path and not a specific json.

Reply:
Check my comment above. We don't mix `--validator-keys` and `--validator-public-keys` otherwise it won't be possible to filter the validators to exit

Reply:
Why can't you filter in the `run` method?

Reply:
Because the validators are mixed when loaded and the `--validator-keys` are supplied as paths (Strings) and not pubkeys.
Theoretically they could be but that would require a paths to pubkeys service